qube/create/templates/lib/lib_groovy/{{ cookiecutter.project_slug }}/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
```

Comment:
This file is a relict.
It uses dependabot version 1.0

The current dependabot file is inside the .github folder.

Please copy it for example from the qoffer2 for an example.

Should also be fixed in all templates.

Reply:
see issue below